packages/backend-defaults/src/entrypoints/httpRouter/httpRouterServiceFactory.ts

```diff
@@ -57,6 +59,7 @@ export const httpRouterServiceFactory = createServiceFactory({
     });
 
     router.use(createAuthIntegrationRouter({ auth }));
+    router.use(createLifecycleMiddleware({ lifecycle }));
```

Comment:
This needs to be hooked up to the `startupRequestPauseTimeout` config right? I think it would be best to pass `config` in here and read it within the creator instead, same as the `createCredentialsBarrier` does

Reply:
Oh, right. Good catch! Just added it back.